wrapper/python/wolfssl/certs/ca-cert.pem

```diff
@@ -0,0 +1,87 @@
+Certificate:
```

Comment:
We should be using the certs from the main ./certs directory to avoid duplication here.  This comment applies to all certs (not just ca-cert.pem).